src/pyflexplot/plots.py

```diff
@@ -6,7 +6,9 @@
 centralize it.
 
 There's tons of nested if-statements etc. because the goal during the cleanup
-phase that included the centralization was to avoid falling into the trap of
+phase was to avoid falling into the trap offile_pathprint(
```

Comment:
In the original comment, the line ends with 'of'. Seems like some random funciton name has been inserted at the end?

Reply:
Indeed, I reversed it.